src/base/os.lua

```diff
@@ -455,12 +455,25 @@
 
 --
 -- Run a shell command and return the output.
+--
+-- @param cmd Command to execute
+-- @param streams Standard stream(s) to output
+-- 		Must be one of 
+--		- "botn" (default)
+--		- "output" Return standard output stream content only
+--		- "error" Return standard error stream content only
 --
 
-	function os.outputof(cmd)
+	function os.outputof(cmd, streams)
 		cmd = path.normalize(cmd)
+		local redirection = " 2>&1"
+		if streams == "output" then
+			redirection = " 2>/dev/null"
+		elseif streams == "error" then
+			redirection = " 2>&1 1>/dev/null"
+		end

```

Comment:
Should this perhaps validate that `streams` is a valid value? I imagine an error would be better than it just falling back to `"both"`?

Reply:
I have updated the PR in this way